src/content/docs/apm/agents/php-agent/installation/php-agent-installation-ubuntu-debian.mdx

```diff
@@ -17,7 +17,7 @@ Our [PHP agent](/docs/agents/php-agent/getting-started/new-relic-php#configurati
 
```

Comment:
Can this be combined with another post? If not, this needs this format

title: 'Insert a title here' 
summary: 'Insert a brief description of the feature' 
releaseDate: 'yyyy-mm-dd' 
learnMoreLink (optional): 'https://example.com' 
getStartedLink (optional): 'https://example.com'